script/Ciel/Day 5/05_01_CIEL05_1D.txt

```diff
@@ -123,57 +147,65 @@ A %{i}normal%{/i} human would go insane! There's no way you could be used to see
 [sha:dec1835ff5e14c27953e6be1d57fd7e8151acb0f]{
 -- Page 161, Offset 11308.
 -- 息が詰まる。
-I have trouble breathing.
+My breath catches in my throat.
 }
 [sha:f0ae687acf1a3f7af8909ce50cd4f8289cd5971a]{
 -- Page 161, Offset 11309.
 -- <女|ノエル>の泣き声は、<感情的|ヒステリック>でありながら、的確に俺の矛盾を指摘していた。
-Noel-sensei's crying voice, hysterical as it sounds, rightly points out the contradiction.
+Noel-sensei may be screaming hysterically,
+ but she's still incisively called out my abnormal behavior.
 }
 [sha:08c04cc9bc248389e164da91cbbcabc58dc1beac]{
 -- Page 162, Offset 11310.
 -- あり得ない空間。
-An impossible space.
+This is an unimaginable space.
 }
 [sha:695fe34d82a2bca1f4bc25c239843b03dca5ecac]{
 -- Page 162, Offset 11311.
 -- 蟻の巣のように張り巡らされた地下墓地。動く死体たち。
-An underground graveyard stretching out like an ant nest. Walking dead bodies.
+A sprawling underground graveyard that resembles some kind of ant nest.
+ Corpses that walk around.
 }
 [sha:e7ecfc86ec84d048d4dcb240896e9d4be95d58ac]{
 -- Page 162, Offset 11312.
 -- 生きながら食われた生存者。
-Survivors eaten alive.
+Survivors being eaten alive.
 }
 [sha:ed660e061f9709e9ab4f6b8ed3c47967cd98997b]{
 -- Page 162, Offset 11313.
 -- 牢屋の中には積み重なった死体の壁。
-A wall of corpses piled up inside the prison.
+Walls of corpses piled high inside the jail cells.
 }
 [sha:c3a75437c6aaf9a60e3cb20e6c53fb072c253315]{
 -- Page 163, Offset 11314.
 -- 見ろ。これが地獄だ。
-Look. This is hell.
+Take a look around.
+ This is hell.
 }
 [sha:e82e11b67957676708d76599a0335e02e0f33698]{
 -- Page 163, Offset 11315.
 -- 命あるものなら正視できない、慣れる事のない暗黒だ。
-This is a darkness the living can never look straight at, nor get used to.
+This is a darkness that the living cannot face head-on,
+ that they can never get used to.
 }
 [sha:3ffc7f2ceafdc2e4971a84a6a397e7b3ba9c493a]{
 -- Page 163, Offset 11316.
 -- 理性ある人間だからこそ、この中で<正気|じぶん>を保つ事はできない。
-It is precisely because humans are rational, that they cannot maintain their sense of self here.
+It is precisely because humans are rational that they cannot maintain their sense of self here.
 }
 [sha:cadf1aa963ab5491f9eabc43b322d12ad8568218]{
 -- Page 163, Offset 11317.
 -- まっとうな人間であるのなら。ここでは皆、<死体|やつら>以上に<知性|あたま>のない動物になるしかない。
-Proper human beings, in this kind of place, have no other choice but to become animals no more intelligent than %{i}them%{/i}.
+Normal human beings, at any rate.
```

Comment:
no comma

Reply:
Hmm, I kinda like the pause here for emphasis.

Reply:
comma is fine for me

Reply:
I'd still rather it drop but I like democracy more than commas so it can stay